### DISCUSSION TITLE
Not able to download from Udemy

### DISCUSSION THREAD

**Discussion by skodilkar** [aclap-dev/video-downloadhelper#1049] [Help and Q&A]
### Your question

I've been using this extension for quite a long time. It was working excellently. But suddenly it stopped downloading the videos from Udemy. It's not showing any error. Can you please advice?

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: en-GB
coapp: {"found":true,"new_version":false,"path":"/home/user/.local/share/vdhcoapp/vdhcoapp","version":"2.0.19"}
license: {"unneeded":true}
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36
{
    "used_sidebar_button": true,
    "download_directory": "/home/user",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
            "__serializer_value": "/home/user/solid_growth"
        }
    },
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 3,
    "successfull_dl": 22
}

**Comment by EmilyMCjava** [comment]
sometimes when a video is not detecting and you press play aswell, refresh the page to see if the problem persists because this fixed it for me. hope this helps!

**Comment by skodilkar** [comment]
Tried all the options. But its saying we are not able to download from this website now

________________________________

**Reply by EmilyMCjava** [reply]
all you can do now if thats happening is to wait for the extension to support udemy (again) if you cant wait spam the devs (dont do that :/)

**Reply by GrampaWildWilly** [reply]
There are quite a few other discussion of Udemy (and Hotmart) in this forum.  You should find them & read them.

**Reply by devinschumacher** [reply]
> There are quite a few other discussion of Udemy (and Hotmart) in this forum. You should find them & read them.

if you're gonna take the time to comment you could've at least taken the time to link or mention the discussion number...

**Comment by devinschumacher** [comment]
+1
